tests/testJobManager.py

```
@@ -0,0 +1,142 @@
+import redis
+from mock import MagicMock, patch
+
+from jobManager import JobManager
+from jobQueue import JobQueue
+from tangoObjects import TangoJob, TangoMachine
+from config import Config
+from preallocator import *
+
+
+class TestJobManager(unittest.TestCase):
+    def createTangoMachine(self, image, vmms, vmObj={"cores": 1, "memory": 512}):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)